charts/generic-govuk-app/templates/deployment.yaml

```
@@ -126,6 +126,10 @@ spec:
             {{- with .Values.appExtraVolumeMounts }}
               {{- . | toYaml | trim | nindent 12 }}
             {{- end }}
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh","-c","sleep 15"]
```

simplifies to
```suggestion
                command: ["sleep", "15"]
```
(similarly below)